scripts/maps/map1.json

```diff
@@ -0,0 +1 @@
+{"tileWidth":64,"tileHeight":100,"width":10,"height":21,"grid":[[0,1,2,3,4,5,6,7,8,9],[10,11,12,13,{"x":1,"y":0.2,"node":1,"map":14},15,16,17,18,19],[20,21,22,23,24,{"x":0.8,"y":0.9,"node":1,"map":25},26,27,28,29],[30,31,32,33,34,35,36,37,38,39],[40,41,42,43,{"x":0.4,"y":0.6,"node":1,"map":44},45,46,47,48,49],[50,51,52,53,54,55,56,57,58,59],[60,61,62,63,64,{"x":0.9,"y":0.7,"node":1,"map":65},66,67,68,69],[70,71,72,73,74,75,76,77,78,79],[80,81,82,83,{"x":0.7,"y":0.7,"node":1,"map":84},85,86,87,88,89],[90,91,92,93,94,95,96,97,98,99],[100,101,102,103,104,{"x":0.9,"y":0.4,"node":1,"map":105},106,107,108,109],[110,111,112,113,114,115,116,117,118,119],[120,121,122,123,{"x":1,"y":0.8,"node":1,"map":124},125,126,127,128,129],[130,131,132,133,134,135,136,137,138,139],[140,141,142,143,144,145,{"x":0.2,"y":0.8,"node":1,"map":146},147,148,149],[150,151,152,153,154,155,156,157,158,159],[160,161,162,163,{"x":0.6,"y":0.7,"node":1,"map":164},165,166,167,168,169],[170,171,172,173,174,175,176,177,178,179],[180,181,182,183,184,185,{"x":0.2,"y":0.6,"node":1,"map":186},187,188,189],[190,191,192,193,194,195,196,197,198,199],[200,201,202,203,204,205,206,207,208,209]]}
```

Comment:
This is the output of the devkit map builder. We export and put it here. It's used as raw data for duplicate map. The final map output and empty map output will be prettified.